src/vfs/public.c

```diff
@@ -160,10 +199,28 @@ bool vfs_mkparents(const char *path) {
 }
```

Comment:
Since `vfs_mkparents_recurse` no longer actually recurses, it makes sense to just merge it back into `vfs_mkparents`. This way you also avoid making a redundant copy of the string.